### PR TITLE
Fedora 5 RC1 docker release

### PR DESCRIPTION
**Pull request by yinlinchen**
# What does this Pull Request do?
Update Fedora 5 RC1 docker release. Re-structure the repo.

# What's new?

1. Put Fedora 4 releases into Fedora4 folder
2. Add Fedora 5 RC1 docker release

# How should this be tested?
Follow the new README.md and test all three
Run Fedora with a file-based objects database (Default Fcrepo 5.0.0-RC-1):
```
# Start server
docker-compose up -d

# Shutdown server
docker-compose down
```

Run Fedora with a MySQL database:
```
# Start server
docker-compose -f fcrepo-mysql.yml up -d

# Shutdown server
docker-compose -f fcrepo-mysql.yml down
```

Run Fedora with a PostgreSQL database:
```
# Start server
docker-compose -f fcrepo-postgres.yml up -d

# Shutdown server
docker-compose -f fcrepo-postgres.yml down
```

Test the Fedora 5 through web browser.

# Interested parties
Tag (@fcrepo4-labs/committers )

### CODE REVIEW

File: Fedora4/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  appnetwork:
+    driver: bridge
+
+services:
+  fcrepo:
+    container_name: fcrepo
+    build:
+      context: ./docker/services/fcrepo
+      args:
+        FEDORA_VERSION: 4.7.5
```
Comment (bseeger) [inline Fedora4/docker-compose.yml:13]: This is great! I'm wondering if we want to stop including versions below... 4.6?  Or just support 4.7.5, though that might be too strict.

Comment (awoods) [reply Fedora4/docker-compose.yml:13]: @bseeger : are you suggesting the possibility of removing the `4.2.0` to `4.6.2` directories:
https://github.com/fcrepo4-labs/fcrepo4-docker
Do you see any overhead in keeping them around?

**Comment by dbernstein** [comment]
@yinlinchen :  I've confirmed the default  compose command works.

**Comment by dbernstein** [comment]
@yinlinchen : I'll merge it if you want - however I currently do not have admin privileges on this repo.

**Comment by awoods** [comment]
Try now, @dbernstein .

**Pull request merged**